docs/api/client.rst

```diff
@@ -0,0 +1,31 @@
+client - communicating with kernels
+===================================
+
+.. module:: jupyter_client.client
```

Comment:
I would put this as top-level in `jupyter_client`, so it's documented as `jupyter_client.KernelClient`. Same for Manager.